### PR TITLE
CFINSPEC-341 Attestation documentation added

### PR DESCRIPTION
**Pull request by Nik08**
## Description

Attestation documentation added
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by Vasu1105 — APPROVED** [review]

File: docs-chef-io/content/inspec/attestations.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+[menu]
+ [menu.inspec]
+   title = "Attestations"
+   identifier = "inspec/reference/attestations.md Attestations"
+   parent = "inspec/reference"
+   weight = 140
++++
+
+Attestations is a mechanism to mark `Not Reviewed (N/R)` tests as `passed` or `failed` manually using an attestations file.
```
Comment (dkumaras) [inline docs-chef-io/content/inspec/attestations.md:14]: 'are' is fine?

**Review by dkumaras — COMMENTED** [review]
Pls check and confirm

**Comment by Nik08** [comment]
@dkumaras I feel it should be "attestations **is** required", rather than **are**. And from line 75 onwards, you have made the list pointers each numbered with "1", is there a reason behind that? 
Everything else looks good :)

**Review by ahasunos — APPROVED** [review]

**Comment by dkumaras** [comment]
> @dkumaras I feel it should be "attestations **is** required", rather than **are**. And from line 75 onwards, you have made the list pointers each numbered with "1", is there a reason behind that? Everything else looks good :)

Rgd, numbering - Yes, when the content is rendered the numbering is automatically taken care of. This is refrain from providing manual numbering and to avoid any errors.

**Review by dkumaras — APPROVED** [review]

**Review by dkumaras — APPROVED** [review]

**Comment by aaronlippold** [comment]
Did we forget to document the how you attest to 'who' is the attestor - aka the human that provided this data

**Comment by aaronlippold** [comment]
an attestation or a waiver need to document 'who' took the action for accountabily

**Pull request merged**